### PR TITLE
PR for Notes-Home layout

### PR DESCRIPTION
**Pull request by manismk**
- Added Notes-app home layout
- [Link](https://deploy-preview-4--key-notes.netlify.app/app/notes)

### CODE REVIEW

File: src/components/navbar/Navbar.jsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import { Link } from "react-router-dom";
+import { Sidebar } from "../";
+import "./navbar.css";
+import { useState } from "react";
+import { Person, Menu } from "@mui/icons-material";
+import {} from "@mui/material";
```
Comment (shraddha-amber) [inline src/components/navbar/Navbar.jsx:6]: Remove empty import

**Review by shraddha-amber — APPROVED** [review]
Clean jsx and good class names! 
Looks good!

**Review by rahulmistry751 — APPROVED** [review]
Clean UI and code

File: src/components/navbar/Navbar.jsx (hunk 1 above)
Comment (manismk) [reply src/components/navbar/Navbar.jsx:6]: will remove it

**Pull request merged**